tasties_app/templates/tasties_app/recipes.html

```diff
@@ -5,9 +5,25 @@
     <link rel="stylesheet" type="text/css" href="{% static 'css/recipes.css' %}" />
 {% endblock %}
 {% block content %}
+    {% block categories_block %}
+        <div class="container-fluid bg-light d-flex justify-content-center" id="category_btn_container">
+            <form method="get" action="{% url 'recipes' %}">
+                <button type="submit" class="btn" name="category" value="remove_filter" id="remove_filter">Remove Filter</button>
```

Comment:
This will redirect to: `/?category=remove_filter`
I think we could make it cleaner by redirecting just to `/` and omitting the category query param altogether.